### ISSUE TITLE
Add new components: LogiX Pack Block and LogiX Unpack Block

### ISSUE THREAD

**Issue by Nytra** [Xlinka/NeosPlus#135]
Basically you would put this component on a slot and it would prevent any LogiX on or under that slot from being packed/unpacked. Would be useful for making custom LogiX nodes and things that would normally break due to being unpacked accidentally.

**Comment by Xlinka** [comment]
For a simple protection the thought is cool but custom logix nodes... we can just implement those so why use the jank method ?

**Comment by Nytra** [comment]
Ah yeah I guess. Would you be able to implement a proper thing for making like custom LogiX functions? Like packing LogiX into a node and being able to have your own inputs and outputs to it etc?

**Comment by Xlinka** [comment]
so your asking for a dynamic Logix Builder ingame or essentially packing logix into a node and that node to do the function which i mean sure its weird the reason i said we could implement those as in we could just make the code in c# for those functions if they are sensible

**Comment by Nytra** [comment]
yeah, it's just a thing that has been requested for a while on regular Neos, and i thought it might be cool to have something that kinda allows that in Neos Plus.

**Comment by art0007i** [comment]
actually pack block could be useful to put on most objects. because if you snap anything with logix onto redprint then click the redprint pack button it will steal all the nodes from the object and pack them with your logix

**Comment by Nytra** [comment]
Not sure if this would be possible without patching the existing pack and unpack code?
